readme.md

```diff
@@ -26,16 +26,19 @@ The Skoap filters can be found currently in the branch called 'skoap-migration'.
 - updates routing rules without downtime, while supporting multiple types of data sources — including
   [etcd](https://github.com/coreos/etcd), [Innkeeper](https://github.com/zalando/innkeeper), static files and
   custom configuration sources
-- can serve as a Kubernetes Ingress implementation in combination with a controller; [see example](https://github.com/zalando-incubator/kube-ingress-aws-controller)
+- can serve as a
+  [Kubernetes Ingress implementation](https://godoc.org/github.com/zalando/skipper/dataclients/kubernetes)
+  in combination with a controller that will route public traffic to
+  your skipper fleet; [see AWS example](https://github.com/zalando-incubator/kube-ingress-aws-controller)
```

Comment:
will it route the public traffic to the skipper fleet or to the cluster?

Reply:
The kube-ingress-aws-controller will add ALB with targetGroup skipper-ingress. Maybe this is missleading?
Any idea how to improve?

Reply:
maybe it's just me. How about:

```
can serve as a
[Kubernetes Ingress implementation](https://godoc.org/github.com/zalando/skipper/dataclients/kubernetes)
in combination with a controller, [see AWS example](https://github.com/zalando-incubator/kube-ingress-aws-controller);
```

...? I think this is enough detail for the readme level.

Reply:
The problem I see is that people have to understand what our kube-ingress-aws-controller does in order to use skipper in their setup. I hope the added words will explain this in short.

Reply:
ok, sounds good